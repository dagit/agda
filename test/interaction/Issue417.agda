module Issue417 where

data _≡_ (A : Set₁) : Set₁ → Set₂ where
  refl : A ≡ A

abstract

  A : Set₁
  A = Set

  unfold-A : A ≡ Set
  unfold-A = refl

-- The result of inferring the type of unfold-A is the following:
--
--   Set ≡ Set
