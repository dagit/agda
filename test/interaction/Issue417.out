(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-info-action "*Type-checking*" "Checking Issue417 (Issue417.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue417.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked")
(agda2-info-action "*Inferred Type*" "A ≡ Set" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "Checked")
(agda2-info-action "*Inferred Type*" "A ≡ Set" nil)
((last . 1) . (agda2-goals-action '()))
